cardano-shell.cabal

```diff
@@ -45,30 +45,28 @@ library
     , specs
   build-depends:
       aeson
-    , base >=4.7 && <5
+    , base >=4.11 && <5
     , binary
     , bytestring
-    , cardano-prelude
     , Cabal
+    , cardano-prelude
     , concurrency
-    -- general util
     , containers
+    , contravariant
+    , dhall
     , directory
     , formatting
     , iohk-monitoring
-    , safe-exceptions
-    , text
-    , transformers
-    , unix
-    -- json & config
-    , contravariant
-    , dhall
-    , ekg-core
     , process
     , QuickCheck
+    , safe-exceptions
+    , stm
     , text
     , transformers
-    , unix
+  if os(windows)
```

Comment:
Good point, the path we were going with was "let's have things working on *nix and we'll deal with Windows later".

Reply:
this is required by the iohk-nix cross-compilation setup in cardano-ledger.